src/templates/zombienet/polkadot.toml

```diff
@@ -0,0 +1,12 @@
+[relaychain]
+default_command = "./zombienet/bin/polkadot"
```

Comment:
make it a dynamic template because `default_command` depends on the user's OS:
- `"./zombienet/bin/polkadot"` for Linux
- `"polkadot"` for MacOS

Same for others

Reply:
@ipapandinas I never had experience with macOS, so can you explain to me how it should work? As I understand, there is no polkadot binary for macOS to download. So macOS users should clone the Polkadot repo and build their own binaries to use it in Zombienet. Why can't they just copy their binaries in ./zombienet/bin/ ?

Reply:
> So macOS users should clone the Polkadot repo and build their own binaries to use it in Zombienet.

Exactly, manual setup is required for MacOS users. The warnings seems enough for now.